lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset.py

```diff
@@ -156,6 +156,10 @@
             - Subnet name
         aliases:
             - subnet
+    load_balancer:
+        description:
+            - Load balancer name
```

Comment:
I know the other options don't have them but a description entry should always be a sentence and end with `.`